r/configure

```diff
@@ -208,22 +208,19 @@ if grep raspbian /etc/os-release >/dev/null 2>&1; then
   PKG_LIBS="-latomic $PKG_LIBS"
 fi
 
-# If libarrow uses the old GLIBCXX ABI, so we have to use it too
-if [ "$ARROW_USE_OLD_CXXABI" ]; then
-  PKG_CFLAGS="$PKG_CFLAGS -D_GLIBCXX_USE_CXX11_ABI=0"
-fi
-
 # Set any user-defined CXXFLAGS
 if [ "$ARROW_R_CXXFLAGS" ]; then
   PKG_CFLAGS="$PKG_CFLAGS $ARROW_R_CXXFLAGS"
 fi
 
+set -ex
+
```

Comment:
For whenever you're done debugging 

```suggestion
```

Reply:
I don't know, wouldn't it be better to keep at least the `set -e` so that errors bail out immediately?

Reply:
By not exiting immediately, we get to the test program step and then the helpful message that points users to the installation guide, which has troubleshooting tips and points them at JIRA. For that and other reasons (CRAN), I don't want to change how this script is handled, nor does it seem necessary for C++17.